lib/logger.js

```diff
@@ -2,7 +2,7 @@ var winston = require('winston');
 
 var logger = new (winston.Logger)({
     transports: [
-        new (winston.transports.Console)({'timestamp':true})
+        new (winston.transports.Console)()
```

Comment:
I think this is the default configuration so you probably don't need to provide anything. Try replacing this with... `var logger = new (winston.Logger);` or something?